README.Rmd

```diff
@@ -8,92 +8,159 @@ params:
    repo_url: https://github.com/Appsilon/shiny.info
 ---
 
+```{r, echo = FALSE, message = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "inst/assets/README_files",
+  out.width = "100%"
+)
+suppressWarnings(library(shiny.info))
+```
+
 <link href="http://fonts.googleapis.com/css?family=Maven+Pro:400,700|Inconsolata" rel="stylesheet" type="text/css">
 <link href='docs/style.css' rel='stylesheet' type='text/css'>
 
-`r params$repo_name`
+shiny.info <a href='https://github.com/Appsilon/shiny.info'><img src='inst/assets/README_files/logo.png' align="right" height="150" /></a>
 ===========
 
 <div class="subheader">
-Display simple diagnostic info of your Shiny app in any of its corners.
+Display simple information of the [shiny](https://shiny.rstudio.com/) project in the user interface of the app.
 </div>
 
-<div class="section level2">
+</br>
 
-What can be displayed there?
+<!-- badges: start -->
+[![CRAN status](https://www.r-pkg.org/badges/version/shiny.info)](https://cran.r-project.org/package=shiny.info)
+[![R build status](https://github.com/Appsilon/shiny.info/workflows/R-CMD-check/badge.svg)](https://github.com/Appsilon/shiny.info/actions?workflow=R-CMD-check)
+[![Codecov test coverage](https://codecov.io/gh/Appsilon/shiny.info/branch/master/graph/badge.svg)](https://codecov.io/gh/Appsilon/shiny.info?branch=master)
+<!-- badges: end -->
 
-* App version
-* Code commit
-* Last data import date
-* ... anything else you find useful for your development
+<div class="section level2">
 
 How to install?
 ---------------
 
-The most recent version you can get from this repo using [devtools](https://github.com/hadley/devtools).
+You can install shiny.info from CRAN repository:
 
 ```
-devtools::install_github("Appsilon/shiny.info")
+install.packages("shiny.info")
 ```
 
-### Example
-
-Diagnostic info like app version can be displayed with function `display()`: `shiny.info::display("My App Version")`. See top right corner here:
 
-![](inst/assets/README_files/example.png)
-
-1. Install `shiny.info`: `devtools::install_github("Appsilon/shiny.info")`
-2. Just run the code below:
+The most recent version you can get from this repo using [devtools](https://github.com/hadley/devtools).
```

Comment:
The most recent version you can get from this repo using [devtools](https://github.com/hadley/devtools).